test/test_unistd.rs

```diff
@@ -624,6 +624,27 @@ fn test_sysconf_unsupported() {
     assert!(open_max.expect("sysconf failed").is_none())
 }
 
+
+#[cfg(target_os = "linux")]
+#[test]
+fn test_getresuid() {
+    let resuid = getresuid().unwrap();
+    // Assume that tests are never run as root
```

Comment:
LOL!  Of course the tests sometimes run as root!  Some tests only work when run as root!